alpha/lib/enums/PermissionName.php

```diff
@@ -236,6 +236,7 @@
 	
 	const FEATURE_SHOW_FLASH_STUDIO = 'FEATURE_SHOW_FLASH_STUDIO';
 	const FEATURE_SHOW_HTML_STUDIO = 'FEATURE_SHOW_HTML_STUDIO';
+	const FEATURE_ENABLE_USAGE_DASHBOARD = 'FEATURE_ENABLE_USAGE_DASHBOARD';
```

Comment:
Yes, in the DB.
This enum contains only few permissions that used internally by the server code.